src/endpoints/seed/index.ts

```diff
@@ -853,47 +849,56 @@ export const seed = async ({
     ),
   )
 
-  payload.logger.info(`— Seeding globals...`)
-
-  await Promise.all([
-    payload.updateGlobal({
-      slug: 'footer',
-      data: {
-        navItems: [
-          {
-            link: {
-              type: 'custom',
-              label: 'Admin',
-              url: '/admin',
-            },
-          },
-          {
-            link: {
-              type: 'custom',
-              label: 'Source Code',
-              newTab: true,
-              url: 'https://github.com/payloadcms/payload/tree/main/templates/website',
-            },
-          },
-          {
-            link: {
-              type: 'custom',
-              label: 'Payload',
-              newTab: true,
-              url: 'https://payloadcms.com/',
-            },
-          },
-        ],
+  payload.logger.info(`— Seeding footers...`)
+  const footerData: Record<Tenant['slug'], {}> = {
+    nwac: {
+      address: '249 Main Ave. S, Suite 107-366\nNorth Bend, WA 98045\n(206) 909-0203',
+      email: 'info@nwac.us',
+      socialMedia: {
+        instagram: 'https://www.instagram.com/nwacus',
+        facebook: 'https://www.facebook.com/NWACUS/',
+        twitter: 'https://x.com/nwacus',
+        linkedin: 'https://www.linkedin.com/company/nw-avalanche-center',
+        youtube: 'https://www.youtube.com/channel/UCXKN3Cu9rnnkukkiUUgjzFQ',
       },
-    }),
-    payload.updateGlobal({
-      slug: 'nacWidgetsConfig',
-      data: {
-        version: '20250313',
-        baseUrl: 'https://du6amfiq9m9h7.cloudfront.net/public/v2',
+    },
+    sac: {
+      address: '11260 Donner Pass Rd. Ste. C1 - PMB 401\nTruckee, CA 96161\n(530) 563-2257',
+      email: 'info@sierraavalanchecenter.org',
+      socialMedia: {
+        instagram: 'https://www.instagram.com/savycenter/',
+        facebook: 'https://www.facebook.com/sacnonprofit',
+        youtube: 'https://www.youtube.com/channel/UCHdjQ0tSzYzzN0k29NaZJbQ',
       },
-    }),
-  ])
+    },
+    snfac: {
+      address: '249 Main Ave. S, Suite 107-366\nNorth Bend, WA 98045\n(206) 909-0203',
+      email: 'info@nwac.us',
+      socialMedia: {},
+    },
+  }
+
+  const footer = (
+    tenant: Tenant,
+    brandImages: Record<Tenant['slug'], Record<string, Media>>,
+  ): RequiredDataFromCollectionSlug<'footer'> => {
+    return {
+      tenant: tenant.id,
+      footerLogo: brandImages[tenant.slug]['logo'].id,
+      name: tenant.name,
```

Comment:
Would we ever want the footer's name to be something other than the tenant's name? If not, we should imo remove the field and just look up the tenant's name for rendering.

Reply:
I am not sure - which is why I made this dynamic. I will ask Rusty and report back.